src/helper.js

```diff
@@ -0,0 +1,25 @@
+export function formatDate(dateString) {
```

Comment:
👍. separating reusable functions is a very good approach to keep things orderly

Reply:
Thank you.